### PR TITLE
px > rem

### PR DESCRIPTION
**Pull request by W3stside**
PX to REM or EM values

@media queries use EM over REM from [here](https://zellwk.com/blog/media-query-units/)

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr369--dexreact.review.gnosisdev.com

**Comment by Velenir** [comment]
:+1: 
Relative units always > absolute units
`px` everywhere remind me of the dark ages of _pixel perfect_ designs

File: src/const.ts [hunk 1]
```diff
@@ -38,12 +38,21 @@ export const LEGALDOCUMENT = {
...
+  // TABLET_LARGE: 866,
+  // WEB_SMALL: 1024,
+
+  // REM SIZES:
+  MOBILE_SMALL: 20,
+  MOBILE: 31.25,
+  MOBILE_LARGE: 33.25,
+  TABLET: 45,
+  TABLET_LARGE: 54.125,
+  WEB_SMALL: 64,
```
Comment (alfetopito) [inline src/const.ts]: Curiosity: how do you get from `px` screen sizes to `em`?

**Comment by anxolin** [comment]
This was fastly merged. Is good to leave some time, or write in our channel if something need to be reviewed faster

File: src/const.ts (hunk 1 above)
Comment (Velenir) [reply src/const.ts]:
```
1rem = 16px
Yrem = Xpx
```
or did you do something more interesting, @W3stside ?

**Pull request merged**